app/webhook.go

```diff
@@ -145,15 +150,26 @@ func (a *App) TriggerWebhook(payload *model.OutgoingWebhookPayload, hook *model.
 	}
 }
 
-func (a *App) doOutgoingWebhookRequest(url string, body io.Reader, contentType string) (*model.OutgoingWebhookResponse, error) {
-	req, err := http.NewRequest("POST", url, body)
+func (a *App) doOutgoingWebhookRequest(url string, body io.Reader, contentType, secretToken string) (*model.OutgoingWebhookResponse, error) {
+	bodyBytes, err := ioutil.ReadAll(body)
+	if err != nil {
+		return nil, fmt.Errorf("failed to read outgoing webhook request payload: %w", err)
+	}
+	bodyReader := bytes.NewReader(bodyBytes)
+
+	req, err := http.NewRequest("POST", url, bodyReader)
 	if err != nil {
 		return nil, err
 	}
 
 	req.Header.Set("Content-Type", contentType)
 	req.Header.Set("Accept", "application/json")
 
+	now := strconv.FormatInt(time.Now().Unix(), 10)
```

Comment:
Existing hooks won't have a `SecretToken` -- should we check for this and omit if empty?